CHANGELOG.md

```diff
@@ -4,10 +4,15 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/). This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
+Nil.
+
+## [0.8.4] - 2021-01-15
+### Added
+- Added support for Ruby 3.0.0. [#53](https://github.com/Shopify/pseudolocalization/pull/53)
+
 ### Changed
 - Switched to Ruby 2.7.1 for development. [#45](https://github.com/Shopify/pseudolocalization/pull/45)
```

Comment:
This can probably be deleted right as it's superseded by #55

Reply:
I was going back on forth on that for a while. Yeah, I think we can drop it.